packages/techdocs-common/src/stages/publish/awsS3.ts

```diff
@@ -47,7 +46,7 @@ export class AwsS3Publish implements PublisherBase {
       );
     }
 
-    const storageClient = new aws.S3({
+    const storageClient = new S3({
       credentials: { accessKeyId, secretAccessKey },
       ...(region && { region }),
```

Comment:
I tried to initialise without any `region` property and the aws client throws an error `Region is missing`

Looking at documentation, it feels that region is required for the aws-sdk v3. It will read the  `AWS_REGION` and `AWS_DEFAULT_REGION` environment variables if not provided here.

Just an FYI, I wanted to share. Will include this bit in https://github.com/backstage/backstage/issues/3946

Reply:
Oh yes you are right !
I dit not pay enough attention to this change introduced in v3.

Do you want me to change it in this PR ?

Reply:
I think it's fine to leave it as it is, and do it in "better AWS credentials handling" PR. :)